### PR TITLE
MB-17001 - Fix pricer operations order

### PR DESCRIPTION
**Pull request by michaelvaldes-truss**
## Summary

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Jira acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/run-storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/about/application-setup/milmove-local-client/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/about/development/testing)

### How to test

1. Access the
2. Login as a
3.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/about/supported-browsers) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/about/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

File: pkg/services/ghcrateengine/domestic_crating_pricer_test.go [hunk 1]
```diff
@@ -16,7 +16,7 @@ const (
 	dcrtTestBasePriceCents       = unit.Cents(2300)
 	dcrtTestEscalationCompounded = 1.125
 	dcrtTestBilledCubicFeet      = unit.CubicFeet(10)
-	dcrtTestPriceCents           = unit.Cents(25875) // dcrtTestBasePriceCents * (dcrtTestBilledCubicFeet ) * dcrtTestEscalationCompounded
+	dcrtTestPriceCents           = unit.Cents(25880)
```
Comment (rogeruiz) [inline pkg/services/ghcrateengine/domestic_crating_pricer_test.go:19]: I've been struggling a bit while reviewing this earlier before our sync meeting. This in-line comment seems like it was trying to be helpful. Now that it's removed, I can only imagine that the calculations don't add up the same way. Are we making the tests conform to our expectations here? I also went ahead and tried to add these together and got a different number than what's shown on line 19.

```
2300 × 10 × 1.125 = 25,875
```

**Review by rogeruiz — COMMENTED** [review]
Just a comment on this while I was reviewing it yesterday/this morning.

File: pkg/services/ghcrateengine/domestic_crating_pricer_test.go (hunk 1 above)
Comment (ruizajtruss) [reply pkg/services/ghcrateengine/domestic_crating_pricer_test.go:19]: I didn't work on this one but I believe that comment would no longer be accurate. I believe we're multiplying those goodies to the escalated price not the base price

**Review by ruizajtruss — APPROVED** [review]
All the changes are consistent with that was expected for the updated order of operations and all tests pass 👍

**Pull request merged**